packages/device_info_plus/device_info_plus/pubspec.yaml

```diff
@@ -26,11 +26,11 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  device_info_plus_platform_interface: ^4.0.1
-  device_info_plus_macos: ^4.0.2
-  device_info_plus_linux: ^4.0.2
-  device_info_plus_web: ^4.0.2
-  device_info_plus_windows: ^5.0.2
+  device_info_plus_platform_interface: ^5.0.0
+  device_info_plus_macos: ^4.0.3
+  device_info_plus_linux: ^4.0.3
+  device_info_plus_web: ^4.0.3
+  device_info_plus_windows: ^5.0.3
```

Comment:
These need to change to major version bumps if the platform interface has a major version bump